docs/postgres_install.rst

```diff
@@ -0,0 +1,86 @@
+===============================
+RDataRetriever with PostgreSQL
+===============================
+
+RDataRetriever supports different DBMS for storing of downloaded datasets through the Retriever R API. It supports PostgreSQL, MySQL, SQLite and MS Access.
+
+
+Installation of PostgreSQL in Ubuntu
+====================================
+
+PostgreSQL installation command differs on the basis of Ubuntu version (Trusty 14.04, Xenial 16.04, Zesty 17.04). The following are the steps:
```

Comment:
I think we can probably stick to 17.04+ at this point since 18.04, the next LTS release, will be out in a couple of weeks and 14.04 and 16.04 shouldn't be too widely used at that point.